log.capnp

```diff
@@ -394,9 +394,10 @@ struct PandaState @0xa7649e2575e4591e {
   ignitionLine @2 :Bool;
   controlsAllowed @3 :Bool;
   gasInterceptorDetected @4 :Bool;
-  canSendErrs @7 :UInt32;
-  canFwdErrs @8 :UInt32;
-  canRxErrs @19 :UInt32;
+  safetyTxBlocked @24 :UInt32;
+  safetyRxInvalid @29 :UInt32;
+  txBufferOverflow @7 :UInt32;
+  rxBufferOverflow @19 :UInt32;
```

Comment:
is this right? look at https://github.com/commaai/panda/pull/1068, it seems like this should be
```
  safetyTxBlocked @24 :UInt32;
  safetyRxInvalid @19 :UInt32;
  txBufferOverflow @8 :UInt32;
  rxBufferOverflow @7 :UInt32;
```